packages/mobile/src/geth/saga.ts

```diff
@@ -98,6 +101,76 @@ export function* waitForGethConnectivity() {
   }
 }
 
+function* pollGethStatusUntilSynced() {
+  let head: BlockHeader = yield select(chainHeadSelector)
+
+  // Head may be null on startup
+  if (!head) {
+    head = (yield take(Actions.SET_CHAIN_HEAD)).head
+  }
+
+  while (true) {
+    if (blockIsFresh(head)) {
+      return true
+    }
+
+    head = (yield take(Actions.SET_CHAIN_HEAD)).head
+  }
+}
+
+export function* waitForGethSync() {
+  const fornoEnabled = yield select(fornoSelector)
+  if (fornoEnabled) {
+    return
+  }
+
+  const { result } = yield race({
+    result: call(pollGethStatusUntilSynced),
+    timeout: delay(GETH_SYNC_TIMEOUT),
+  })
+
+  if (!result) {
+    ValoraAnalytics.track(GethEvents.geth_sync_timeout)
```

Comment:
Why track this event independently? We should get an error event from whichever flow fails due to this timeout, and we can get sync timing information from the `network_sync_*` events, so it's unclear to me what additional information this event provides.

Reply:
I agree...will remove this